src/main/java/uk/gov/ons/census/casesvc/service/BlankQuestionnaireService.java

```diff
@@ -0,0 +1,178 @@
+package uk.gov.ons.census.casesvc.service;
+
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.CONT_FORM_TYPE;
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.HH_FORM_TYPE;
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.IND_FORM_TYPE;
+import static uk.gov.ons.census.casesvc.utility.FormTypeHelper.mapQuestionnaireTypeToFormType;
+import static uk.gov.ons.census.casesvc.utility.MetadataHelper.buildMetadata;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.casesvc.model.dto.ActionInstructionType;
+import uk.gov.ons.census.casesvc.model.dto.EventTypeDTO;
+import uk.gov.ons.census.casesvc.model.dto.Metadata;
+import uk.gov.ons.census.casesvc.model.entity.Case;
+import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
+
+@Component
+public class BlankQuestionnaireService {
+
+  private final CaseService caseService;
+
+  private Map<BlankQuestionnaireService.Key, BlankQreRule> rules = new HashMap<>();
+
+  public BlankQuestionnaireService(CaseService caseService) {
+    this.caseService = caseService;
+    setUpRules();
+  }
+
+  private void setUpRules() {
```

Comment:
either have this as a static method or use a static initialiser block like in other places in the Case Processor